### PR TITLE
Adds a multiplication function and tests

### PR DESCRIPTION
**Pull request by kateca0**
I have added  the multiplication functions and test. Kindly close this issue number. thanks

### CODE REVIEW

**Review by kateca0 — COMMENTED** [review]
cool

File: pythoncalculator/multiply.py
Comment (gachiethomas) [inline pythoncalculator/multiply.py:1]: Change the multiplication function from 

```
def multiply(x, y):
    return x + y
```

to 

```
def multiply(x, y):
    return x * y
```

File: pythoncalculator/multiply.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def multiply(x, y):
+    return x + y
```
Comment (ocarlile) [inline pythoncalculator/multiply.py]: Typo in the multiply function.
```suggestion
    return x * y
```

Comment (kateca0) [inline pythoncalculator/multiply.py]:
```suggestion
    return x * y
```
This should have been multiplication not addition

**Review by ocarlile — CHANGES_REQUESTED** [review]

**Review by gachiethomas — APPROVED** [review]
Looks good

**Review by ocarlile — APPROVED** [review]
Looks good, thanks for correcting

**Review by kateca0 — COMMENTED** [review]
GREAT

**Pull request merged**